nevergrad/optimization/recorded_recommendations.csv

```diff
@@ -46,7 +46,7 @@ OnePointDE,0.5018723037790331,0.138846177156,0.640761113342,0.527332665111,,,,,,
 OptimisticDiscreteOnePlusOne,0.7531428339492002,0.0,0.0,1.095956118009,,,,,,,,,,,,
 OptimisticNoisyOnePlusOne,0.0,0.0,0.0,0.0,,,,,,,,,,,,
 PCEDA,0.0,0.0,0.0,0.0,,,,,,,,,,,,
-PSO,0.3540566088595821,-0.367269962925,-1.644114353981,-0.721304358121,6.361340889697,6.361340889697,0.70044955164,-6.361340902404,-0.832318899904,1.390267483003,,,,,,
+PSO,-0.1160475433749192,-0.643267540662,0.932347027466,-1.023530429661,6.361340889697,6.361340889697,0.614048950193,-6.361340902404,-0.355404740174,-0.485783193197,,,,,,
```

Comment:
This modification is only due to the reordering of the random generation of `rp` and `rg` values.